include/deal.II/distributed/tria.h

```diff
@@ -383,6 +383,9 @@ namespace parallel
       virtual void
       clear() override;
 
+      bool
+      is_multilevel_hierarchy_constructed() const override;
```

Comment:
This looks good to me, but can you add documentation to the function?

Reply:
Ah, never mind -- this is the derived class, which inherits the documentation.

Reply:
Done.